docs/docs/filters/filtering.md

```diff
@@ -1,9 +1,9 @@
 # Tracing Event Filtering
 
 ```console
-sudo ./dist/tracee --scope help
-sudo ./dist/tracee --events help
-sudo ./dist/tracee --scope xxx --events xxx
+./dist/tracee man --help
```

Comment:
This does not make any sense to me. I have never seen any userland tool that had "man" together with "--help". Also, having a "man" command without dashes, and a subcommand with dashes is very weird.

Also, man without "--help" has the same output, so why not to just leave "tracee man" here ?

Reply:
`--help/-h` flags are automatic for any command in cobra. I changed the man command to print the same as the -h flag if it doesn't contain any other arg. So, `tracee man` = `tracee man -h`: https://github.com/aquasecurity/tracee/pull/3572/files#diff-35ddad12850bedf904c52eb37d70d82187698a3531c352b2dcd62e6e43447cfaR37-R41

> Also, having a "man" command without dashes, and a subcommand with dashes is very weird.

It is the main logic of the cobra and GNU flags, if it has dashes it is a flag, not a command/subcommand.

> Also, man without "--help" has the same output, so why not to just leave "tracee man" here ?

I'll change the doc, removing the --help flag.